client/blocks/login/login-form.jsx

```diff
@@ -30,13 +34,16 @@ export class LoginForm extends Component {
 		redirectTo: PropTypes.string,
 		requestError: PropTypes.object,
 		translate: PropTypes.func.isRequired,
+		isFormDisabled: PropTypes.bool,
```

Comment:
What's the difference between the `isFormDisabled` prop and the `isDisabled` state? They seem to refer to the same thing, which means one of them (probably `isDisabled`) should go away.